### ISSUE TITLE
Tile 9763 on 2024-10-07 is unsure - large EFFTIME on petals 8,9,0

### ISSUE THREAD

**Issue by dstndstn** [desihub/desisurveyops#226]
This was right after a big slew from SGC to NGC.  Klaus reports a large hexapod and focus adjustment.
Note that the discrepant petals have *larger* efftime.

Anand's plot of the THRUFRAC is attached.

![1](https://github.com/user-attachments/assets/4113a697-cbf3-4a65-8f2a-4dadb77596cb)
![2](https://github.com/user-attachments/assets/77ce7f7b-c1e1-4e71-b5ec-b414234f748a)

**Comment by dstndstn** [comment]
Tile 9795 on 2024-10-11, expid 257567, has odd jumps in efftime between petals
![1](https://github.com/user-attachments/assets/bfc80a87-086a-46ef-b7ed-146e7173fae4)

**Comment by dstndstn** [comment]
Tile 3510 on 2024-10-11, expid 257565, has a sharp break in efftime.
This was the first exposure after the SGC-to-NGC slew.
![1](https://github.com/user-attachments/assets/d2721693-76f8-4c7d-b31b-8f12c3661907)

**Comment by sybenzvi** [comment]
Similar to exp 256540, tile 6643 (2024-10-05) noted by @SGontcho (see #225).

**Comment by geordie666** [comment]
Here's the tile from issue #225. We'll consider these problematic slew/focus tiles together and make a policy at a later date.

![tile-qa-6643-thru20241005](https://github.com/user-attachments/assets/8ebc6de8-9802-47a6-b44c-e8cdb9c3293f)

**Comment by dkirkby** [comment]
Here are the PSFs measured from the acquisition image (by the ETC) for the exposures mentioned above. They all have good seeing and I don't see signs of de-focus at the start of the exposure. Perhaps there is a trend of elongation in GUIDE7,8.

![etc-00256540](https://github.com/user-attachments/assets/0d006db7-4556-4516-9c48-d69edd74b1d4)
![etc-00257565](https://github.com/user-attachments/assets/2e9b6c4a-b9fd-4774-b1e7-4b6b00cedf35)
![etc-00257567](https://github.com/user-attachments/assets/fa1dd13c-3b73-4522-807c-86d817de3255)
![etc-00256827](https://github.com/user-attachments/assets/9ae0bc35-8d32-4e4a-8846-5f4ac9cbf750)

**Comment by dkirkby** [comment]
Btw, these images are stored with the raw data with URLs like https://data.desi.lbl.gov/desi/spectro/data/20241005/00256540/etc-00256540.png

PSFs are drawn in local GFA pixel coordinates and the blue arrow points north.

**Comment by sybenzvi** [comment]
Since we are not hurting for time in the SGC and edges of the NGC, the consensus is to mark these tiles as bad and reobserve them while we can.

Folks on daily ops this week (@araichoor and @djschlegel), please note similar large EFFTIMEs and jumps in this ticket if you should find any. We'll raise the issue with instrument ops if it appears to be a recurring pattern.

**Comment by crockosi** [comment]
I looked at all the telemetry I could find for all four of these tiles.  My summary is I think the telescope is struggling to keep up and keep the objects on target. These tiles are at large HA in the east, so it has to push up against gravity. For the series of tiles we get after these big slews, the first has mean RA guide residuals of > 0.15 arcsec, and the subsequent tiles are at smaller (closer to 0) HA and the guide residuals drop steadily toward zero with each tile at smaller HA. I can't explain the pattern in the sense of which petals have bad flux calibrations, but I didn't try very hard. I looked at the focus, it takes a few minutes to catch up but that is because the boxcar averaging is simple, but all the analysis is fine and it behaves as expected. The hexapod moves in x,y and tip/tilt happen at the start of the slew, which worries me a little and I will look into that more, but that does not seem to be the main effect.

**Comment by daqiii** [comment]
Why does this worry you:
The hexapod moves in x,y and tip/tilt happen at the start of the slew, which worries me a little and I will look into that more, but that does not seem to be the main effect.

We have been operating like this forever. All alignment (slew, ADC, hexapod adjustment based on the LUT) is done in parallel at the start of exposure processing
So why does this worry you?

Klaus

From: ConnieR ***@***.***>
Date: Thursday, October 24, 2024 at 7:40 PM
To: desihub/desisurveyops ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [desihub/desisurveyops] Tile 9763 on 2024-10-07 is unsure - large EFFTIME on petals 8,9,0 (Issue #226)
I looked at all the telemetry I could find for all four of these tiles. My summary is I think the telescope is struggling to keep up and keep the objects on target. These tiles are at large HA in the east, so it has to push up against gravity. 


I looked at all the telemetry I could find for all four of these tiles. My summary is I think the telescope is struggling to keep up and keep the objects on target. These tiles are at large HA in the east, so it has to push up against gravity. For the series of tiles we get after these big slews, the first has mean RA guide residuals of > 0.15 arcsec, and the subsequent tiles are at smaller (closer to 0) HA and the guide residuals drop steadily toward zero with each tile at smaller HA. I can't explain the pattern in the sense of which petals have bad flux calibrations, but I didn't try very hard. I looked at the focus, it takes a few minutes to catch up but that is because the boxcar averaging is simple, but all the analysis is fine and it behaves as expected. The hexapod moves in x,y and tip/tilt happen at the start of the slew, which worries me a little and I will look into that more, but that does not seem to be the main effect.

**Comment by crockosi** [comment]
And I agree we have done fine in that mode.  I have no evidence it is not fine, and I am not asking for any changes. It is on my list to look more closely at the computed corrections to see if the LUT needs improving, and to see if we get systematically different correction values if we get to different locations in alt,az by short vs. long slews. But nothing to report right now.

**Comment by daqiii** [comment]
Yes, checking the LUT is a good idea. Maybe things changed over the years. However, the LUT is queried by target position without any knowledge of the  original position  so it doesn’t know whether the slew is long or short.
Klaus

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by geordie666** [comment]
@akremin, @abhi0395 or @abrodze: Can we set the exposures for the following tile/night combinations to bad and reprocess, please?

`3510` from night `20241011`
`6643` from night `20241005`
`9763` from night `20241007`
`9795` from night `20241011`

**Comment by abhi0395** [comment]
@geordie666 I have marked these tiles as bad and re-ran tsnr and nightqa for these nights.

**Comment by sybenzvi** [comment]
Tile 8769 from 20241030 (exp 260441) has a vaguely similar pattern to these other tiles (low EFFTIME in two petals after a long slew from SGC to NGC). The effect is not as severe. I'm marking this as unsure so we discuss it, but we could pass it.
<img width="1233" alt="Screenshot 2024-10-31 at 3 03 29 PM" src="https://github.com/user-attachments/assets/a696bff5-9bbf-4a3a-8a57-2f213d30263a">

**Comment by schlafly** [comment]
Maybe we should make the rule based on *RMS at a value of ~0.04, which would not catch this case but would catch most of the others.  I would keep this tile, but I would have kept many of the other ones, too.  I think there's an alert if *RMS is greater than 0.1, so much larger than that newly suggested "extra scrutiny after large slew" limit.

I would suggest breaking this into a separate ticket because tracking the reprocessing of the earlier tiles will get confusing if we continue to add tiles to this ticket.

**Comment by geordie666** [comment]
I think Eddie suggests a reasonable metric. My intention is to close the existing tiles in this ticket out later today and then I'll close this ticket. I think we'll just keep an eye out for large slews leading to strong `EFFTIME` patterns across a tile. The instrument team are aware of the tiles in this issue and may (or may not) choose to study improvements after large slews. 

But, I don't know that this hasn't happened before and I think we can deal with it on a case-by-case basis.

**Comment by sybenzvi** [comment]
Agreed, this sounds like a good rule of thumb. We can open another ticket about this, perhaps to discuss where to document the problem. I hate to add more information to the long DailyOps wiki but that's the best I can come up with at the moment...

**Comment by araichoor** [comment]
if useful, I ve done a small "follow-up" of https://github.com/desihub/desisurveyops/issues/245#issuecomment-2456396175:

in these two (large, ~100M) pdf files, I generate similar plots for all main{bright,dark} exposures with 100<ra<120 and 40<dec<60, and efftime_spec>0:
https://data.desi.lbl.gov/desi/users/raichoor/tmpdir/tmp-bright.pdf
https://data.desi.lbl.gov/desi/users/raichoor/tmpdir/tmp-dark.pdf
(I pick the data from kibo if before 20240409, else from daily).

from a quick browsing of those, I couldn t find a clear correlation with the moon, or the ha, or the slew angle...
in the coming days, I can try to make a per-exposure metric (like the dispersion in the sky values over the 5000 fibers), which could be used to explore potential correlations with observational quantities?

**Comment by geordie666** [comment]
@forero: I set the QA for the 5 tiles mentioned in this thread (`3510`, `6643`, `8769`, `9763`, `9795`) back to `none`. They will now reappear in regular tile QA. Can you please mark these tiles as good when you run QA tomorrow, if you beat me to it?

Also, can you report back here when you've done that so I can close this issue (I want to open a new ticket about these general cases rather than about the 5 specific tiles).

**Comment by forero** [comment]
The tile QA looks fine. I marked them as good.

**Comment by geordie666** [comment]
Thanks. I'll open a more general issue about these sorts of tiles soon.
